pranama-ui/src/services/malabar/axios.js

```diff
@@ -0,0 +1,12 @@
+import axios from 'axios'
+
+const apiClient = axios.create({
```

Comment:
Where are you using this function in the code ? I see there are two axios.js

Reply:
Each /axios.js configures a custom axios instance that are used in the services, eg in `pranama-ui/src/services/api/UserService.js` in line 1 you are importing that configured axios instance